### PR TITLE
add convered geometry of juelich_test

### LINKED ISSUE #1092: Convert geometry of tests to WKT
The old geometries used in the [Jülich Tests](https://github.com/JuPedSim/jpscore/tree/master/systemtest/juelich_tests)  need to be converted to WKT (similar to https://github.com/JuPedSim/jpscore/issues/1071).

### PR DESCRIPTION
**Pull request by ChristianHirt**
fixes #1092 
Adds wkt of Juelich_Test  geometries to geometry examples

### CODE REVIEW

**Comment by ChristianHirt** [comment]
test 1 to 5 have a different format but are converted correct
test 8 is seemingly correct but should be reviewed in visdbg
test 6 has a very strange format mixing specifing every point and specifing lines
tests 11, 12 have zero-width walls and can not automatically converted by tool

**Comment by ChristianHirt** [comment]
all files added
wall-width might need to be adapted
i still can´t check the files in visdbg but plot looks good

**Review by Ozaq — COMMENTED** [review]
`evacuation_time.wkt`, `test1.wkt` and `test4.wkt` are identical. Please keep one and delete the other two.

Please rename the file to something like: "rectangular_room.wkt" or similar. Lets help potential users by making things a bit more descriptive.

File: examples/geometry/juelich_tests/test6.wkt [hunk 1]
```diff
@@ -0,0 +1 @@
+GEOMETRYCOLLECTION (POLYGON ((0 4, 2 4, 4 4, 6 4, 7 4, 11 4, 11 0, 7 0, 6 0, 4 0, 2 0, 0 0, 0 4)))
```
Comment (Ozaq) [inline examples/geometry/juelich_tests/test6.wkt:1]: Please rewrite the polygon with just the 4 corner points.

File: examples/geometry/juelich_tests/test8.wkt [hunk 2]
```diff
@@ -0,0 +1 @@
+GEOMETRYCOLLECTION (POLYGON ((10 0, 0 0, 0 4, 10 4, 10 0), (5 0.1, 6 1, 7 0.1, 5 0.1), (2.78907 3.9333, 4.48158 3.9333, 3.75451 2.90826, 2.78907 3.9333)))
```
Comment (Ozaq) [inline examples/geometry/juelich_tests/test8.wkt:1]: Please remove the small gaps between wall and obstacle. We do not support accessible areas which are too small to let agents pass.

Screenshot 2023-04-21 at 15 35 07

File: examples/geometry/juelich_tests/test12.wkt [hunk 3]
```diff
@@ -0,0 +1 @@
+GEOMETRYCOLLECTION (POLYGON ((5.42159 4.73707, 8.42159 4.73707, 8.42159 4.83707, 5.42159 4.83707, 5.42159 6.78707, 8.42159 6.78707, 11.4216 6.78707, 11.4216 2.78707, 8.42159 2.78707, 5.42159 2.78707, 5.42159 4.73707), (8.64584 5.02285, 8.64584 4.78707, 11.3378 4.78707, 11.3378 5.02285, 8.64584 5.02285)))
```
Comment (Ozaq) [inline examples/geometry/juelich_tests/test12.wkt:1]: Screenshot 2023-04-21 at 15 38 50

Both gaps here are too narrow and not supported right now. Can you please ask @chraibi / @JetteSchumann what the intent was with this geometry?

Maybe we can change the geometry and still can test as intended. If we cannot figure out what the idea was just drop this file.

**Review by Ozaq — CHANGES_REQUESTED** [review]

**Comment by Ozaq** [comment]
Please try to give each file a "proper" name that at least tries to describe the geometry

File: examples/geometry/juelich_tests/test6.wkt (hunk 1 above)
Comment (ChristianHirt) [reply examples/geometry/juelich_tests/test6.wkt:1]: this room is a rectangle with the size(11 x 4).
the rooms evacuation_time / test1 / test4 are rectangles of the size (12 x 4).
I do not think we need to keep this geometry.

File: examples/geometry/juelich_tests/test8.wkt (hunk 2 above)
Comment (ChristianHirt) [reply examples/geometry/juelich_tests/test8.wkt]: done

File: examples/geometry/juelich_tests/test6.wkt (hunk 1 above)
Comment (Ozaq) [reply examples/geometry/juelich_tests/test6.wkt:1]: Lets remove it

File: examples/geometry/juelich_tests/test12.wkt (hunk 3 above)
Comment (Ozaq) [reply examples/geometry/juelich_tests/test12.wkt:1]: What's with this one? Did you @ChristianHirt talk to @JetteSchumann or @chraibi ?

Comment (ChristianHirt) [reply examples/geometry/juelich_tests/test12.wkt:1]: It seems that this file was used to test if the pedestrian can see the door even if there is an obstacle. We decided that this geometry has no further use for us and can be removed.

**Pull request merged**